app/server/src/features/accounts/resolver.integration.test.ts

```diff
@@ -0,0 +1,69 @@
+import bcrypt from 'bcryptjs';
+import { getOrCreateServer } from '@local/core/server';
+import { getPrismaClient } from '@local/core/utils';
+import { createMercuriusTestClient } from 'mercurius-integration-testing';
+import * as plugins from '@local/core/plugins';
+import { redisEmitter } from '@local/core/plugins/mercurius';
+import * as jwt from '@local/lib/jwt';
+import { toGlobalId } from '../utils';
+
+const server = getOrCreateServer();
+const testClient = createMercuriusTestClient(server);
+const prisma = getPrismaClient(server.log);
+
+const toUserId = toGlobalId('User');
+
+const userData = {
+    id: '4136cd0b-d90b-4af7-b485-5d1ded8db252',
+    email: 'sallySmith@test.com',
+    firstName: 'Sally',
+    lastName: 'Smith',
+    password: 'testPassword',
+    preferredLang: 'EN',
+};
+
+beforeAll(async () => {
+    // Attack relevant plugins
+    plugins.attachCookieTo(server);
+    plugins.attachMercuriusTo(server);
+
+    // hash password as they are never stored in plaintext on the DB
+    const encryptedPassword = await bcrypt.hash(userData.password, 10);
```

Comment:
I'm still confused as to why we're encrypting the password here.  I don't believe any me query checks if passwords match.  It's simply decoding the jwt.

Reply:
There are checks that do verify a password matches a hash. loginWithPassword for example does a validation check that compares the password to the hashed result. If you don't hash it it will fail that validation during that test.

Reply:
Oh, if I'm understanding correctly, the hashing is not necessary right now, but when we add other resolve tests they will be?  `loginWithPassword` is a mutation IIRC.

Reply:
bump

Reply:
yes that's right, it is necessary for some but not all resolvers.